docs/sources/next/javascript-api/k6-experimental/browser/request/resourcetype.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'resourceType()'
+description: 'Browser module: Request.resourceType method'
+---
+
+# resourceType()
+
+Contains the request's resource type as it was perceived by the rendering engine. ResourceType will be one of the following: `document`, `stylesheet`, `image`, `media`, `font`, `script`, `texttrack`, `xhr`, `fetch`, `eventsource`, `websocket`, `manifest`, `other`.
```

Comment:
> The resource type will be one of the following: ...

As we're in the `resourceType` context and typed it as "resource type" in the first sentence.

Reply:
"**The** resource type will be ...?

Reply:
```suggestion
Contains the request's resource type as it was perceived by the rendering engine. It will be one of the following: `document`, `stylesheet`, `image`, `media`, `font`, `script`, `texttrack`, `xhr`, `fetch`, `eventsource`, `websocket`, `manifest`, `other`.
```

WDYT?

Reply:
OK 😂

Reply:
Changed in f8ae9572bb7b0d50a0d25132a227d94cfd051cbe